fuzz/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address,fuzzer")
```

Comment:
Don't do that, use target_compile_options()!

Reply:
Example: https://gitlab.com/libssh/libssh-mirror/blob/master/tests/fuzz/CMakeLists.txt

Reply:
Got it. will fix.

Reply:
done